config/config.go

```diff
@@ -51,7 +51,8 @@ var (
 )
 
 var (
-	_devEnv = "development"
+	_devEnv  = "development"
+	_testEnv = "test"
```

Comment:
interesting. did go compiler sneeze while processing this line? I thought unused vars were an error